codersdk/deployment.go

```diff
@@ -530,3 +545,35 @@ func (c *Client) AppHost(ctx context.Context) (AppHostResponse, error) {
 	var host AppHostResponse
 	return host, json.NewDecoder(res.Body).Decode(&host)
 }
+
+type WorkspaceConnectionLatencyMS struct {
+	P50 float64
+	P95 float64
+}
+
+type DeploymentStats struct {
+	// AggregatedFrom is the time in which stats are aggregated from.
+	// This might be back in time a specific duration or interval.
+	AggregatedFrom time.Time `json:"aggregated_since" format:"date-time"`
+	// CollectedAt is the time in which stats are collected at.
+	CollectedAt time.Time `json:"collected_at" format:"date-time"`
+	// RefreshingAt is the time when the next batch of stats will
+	// be refreshed.
+	RefreshingAt time.Time `json:"refreshing_at" format:"date-time"`
+
+	PendingWorkspaces  int64 `json:"pending_workspaces"`
+	BuildingWorkspaces int64 `json:"building_workspaces"`
+	RunningWorkspaces  int64 `json:"running_workspaces"`
+	FailedWorkspaces   int64 `json:"failed_workspaces"`
+	StoppedWorkspaces  int64 `json:"stopped_workspaces"`
+
+	WorkspaceConnectionLatencyMS WorkspaceConnectionLatencyMS `json:"workspace_connection_latency_ms"`
+
+	SessionCountVSCode          int64 `json:"session_count_vscode"`
+	SessionCountSSH             int64 `json:"session_count_ssh"`
+	SessionCountJetBrains       int64 `json:"session_count_jetbrains"`
+	SessionCountReconnectingPTY int64 `json:"session_count_reconnecting_pty"`
+
+	WorkspaceRxBytes int64 `json:"workspace_rx_bytes"`
+	WorkspaceTxBytes int64 `json:"workspace_tx_bytes"`
```

Comment:
These could be under `workspaces.rx_bytes`. Singular vs plural (workspace, workspaces) above is a bit confusing currently.